gatsby-node.js

```diff
@@ -51,7 +152,10 @@ const outdatedMarkdownPages = [
   "/what-is-ethereum/",
 ]
 
-exports.onCreateNode = ({ node, getNode, actions }) => {
+// Loops through all the files dictated by Gatsby (building pages folder), as well as
+// folders flagged through the gatsby-source-filesystem plugin in gatsby-config
+// This seems to be where the source and transform holdup is?
```

Comment:
I would suggest removing the comments with questions from the code. I think we can discuss them off record than leaving them there.

And regarding the question, hmm not sure. I think it is more related with the queries execution, when gatsby is preparing to build the pages.